### PR TITLE
#30 introduces example of sourcing investing_com data and exporting

### PR DESCRIPTION
**Pull request by j-dabrowski**
Base PR

### CODE REVIEW

**Comment by j-dabrowski** [comment]
Yes.

**Comment by j-dabrowski** [comment]
Yes ready. Noted

File: examples/kwil/copy_investing.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+# Name
+PIPELINE_NAME = "Generate USD"
+
+
+ROUNDING = 6
+TABLES = [
+    'com_investing_dutch_ttf_natural_gas_futures',
+    'com_investing_usd_eur_historical'
+]
```
Comment (zolotokrylin) [inline examples/kwil/copy_investing.py:36]: How does these tables relate to the USD/CNY?

Comment (zolotokrylin) [reply examples/kwil/copy_investing.py:36]: @j-dabrowski

Comment (j-dabrowski) [reply examples/kwil/copy_investing.py:36]: They don't, am correcting this
Is the goal to collect multiple varieties of lithium, eg. lithium 99.9, 99, 60?

https://www.investing.com/commodities/99.9-min-china-lithium-metal-futures-historical-data
https://www.investing.com/commodities/lithium-carbonate-99-min-china-futures-historical-data
https://www.investing.com/commodities/lithium-cobalt-oxide-60-min-china-futures-historical-data

Comment (j-dabrowski) [reply examples/kwil/copy_investing.py:36]: todo: 
find the endpoint to return prices for each of these commodities
have pipeline details source the data using these endpoints

Comment (j-dabrowski) [reply examples/kwil/copy_investing.py:36]: @zolotokrylin

Comment (zolotokrylin) [reply examples/kwil/copy_investing.py:36]: @joequant

**Pull request merged**